Makefile

```diff
@@ -1,11 +1,16 @@
-init:
+init: setup-git
 	which brew || /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 	brew bundle
 	rbenv install --skip-existing
 	rbenv exec gem update bundler
 	rbenv exec bundle update
 	cd Samples/TrendingMovies && carthage update --use-xcframeworks
 
+setup-git:
+ifneq (, $(shell which pre-commit))
+	pre-commit install
```

Comment:
This setup won't work on new systems. We need to add `pre-commit` to `Brewfile`, otherwise it'll never be installed, and `setup-git` will never install the hooks. Also, I think it'd be better to avoid the dependency indirection and `which` check, by putting `pre-commit install` directly into `make init` after `brew bundle`.

Reply:
It got added to the Brewfile in https://github.com/getsentry/sentry-cocoa/pull/1947